cmake/scripts/FindLIBTOMCRYPT.cmake

```diff
@@ -7,19 +7,11 @@
 # LIBTOMCRYPT_INCLUDE_DIRS - The LIBTOMCRYPT include directories
 # LIBTOMCRYPT_LIBRARIES - The libraries needed to use LIBTOMCRYPT
 
-if(${LIBTOMCRYPT_INCLUDE_DIR})
```

Comment:
You have only left find_path, which I believe will not work on Windows. Unless CMAKE_PATH is provided. Can this change be reverted?
File itself was renamed from FindLibTomCrypt.cmake to FindLIBTOMCRYPT.cmake
Is this change really required?

Reply:
With the original code, cmake was unable to find libtomcrypt unless cmake was invoked a second time without deleting the build directory. I have changed it so that it builds for me on the first try and I think preserves what the original code was trying to do.
https://github.com/cuevavirus/psvpfstools/blob/94471aefbc4e991bc2981f1489237df9c7ff44f2/cmake/scripts/FindLIBTOMCRYPT.cmake#L10-L22

The rename was to match case in ConfigureDependencies.cmake for case-sensitive environments. Alternatively, I could revert the file rename and change the names in ConfigureDependencies.cmake instead.

Reply:
Everything is perfect now. Find script was indeed a mess.